### DISCUSSION TITLE
webui界面优化建议

### DISCUSSION THREAD

**Discussion by bobandbrony** [NapNeko/NapCatQQ#618] [Show and tell]
### 系统版本

乌班图22.04

### QQNT 版本

忘了（应该是最新版 ）

### NapCat 版本

4.2.28

### OneBot 客户端

无

### 发生了什么?

面板基础信息太简陋了

### 如何复现

访问http://(ip):(port)/webui/#/dashboard/basic-info
![image](https://github.com/user-attachments/assets/d496a537-0aa4-4467-9a7d-91dfaf9bd884)

### 期望的结果?

建议在“基础信息”加个当前登录qq信息（包含qq号，qq头像，qq状态（比如在线，隐身之类的）显示，有多少群和好友，其他设备登录情况等），各种连接方式的调用次数统计，显示当前napcat版本
“关于我们”增加检查更新功能，可以的话加个一键更新

**Comment by bietiaop** [comment]
两次你把 `feat` 写成 `Fear`，厉害。

**Comment by bobandbrony** [comment]
> 两次你把 `feat` 写成 `Fear`，厉害。

第一次原本是[bug] wenui..... ，后来被[MliKiowa](https://github.com/MliKiowa)改成了[Fear]，然后我以为就是提建议就这样写了。现在[MliKiowa](https://github.com/MliKiowa)已经改正确了

**Comment by MliKiowa** [comment]
> > 两次你把 `feat` 写成 `Fear`，厉害。
> 
> 第一次原本是[bug] wenui..... ，后来被[MliKiowa](https://github.com/MliKiowa)改成了[Fear]，然后我以为就是提建议就这样写了。现在[MliKiowa](https://github.com/MliKiowa)已经改正确了

非常抱歉 由于是晚上 处理比较着急 所以当时改错了

**Comment by bietiaop** [comment]
笑死我了

**Comment by bobandbrony** [comment]
> > > 两次你把 `feat` 写成 `Fear`，厉害。
> > 
> > 
> > 第一次原本是[bug] wenui..... ，后来被[MliKiowa](https://github.com/MliKiowa)改成了[Fear]，然后我以为就是提建议就这样写了。现在[MliKiowa](https://github.com/MliKiowa)已经改正确了
> 
> 非常抱歉 由于是晚上 处理比较着急 所以当时改错了

没事，还有就是改下标签吧，改成enhancement
此外建议在新建问题(new issues)再增加一个选项，改进建议

**Comment by sisi0318** [comment]
建议建议的人pr

**Comment by pk5ls20** [comment]
> 此外建议在新建问题(new issues)再增加一个选项，改进建议

好的

**Comment by bietiaop** [comment]
- [x] 在“基础信息”加个当前登录qq信息（包含qq号，qq头像，qq状态（比如在线，隐身之类的）显示
- [ ] 有多少群和好友，其他设备登录情况等）
- [ ] 各种连接方式的调用次数统计
- [x] 显示当前napcat版本
- [x] “关于我们”增加检查更新功能 `已经在首页添加`
- [ ] 可以的话加个一键更新 `TODO`

# Additon

- [x] 系统状态和其他的版本信息

**Reply by Shua-github** [reply]
有头就可以看了
